Modification of IO project 5 (#45)
IntersectMBO/budget-management: metadata/2026-04-02-modification-io-5/README.md

# 2026-04-07 Modify Input Output Engineering Core Development Proposal (5of6)

## Transaction ID : ``

## Transaction Inputs

The UTxO marked `UTXO-EC-0002-25-05`

#### UTXO:`b49d27563e6757ff1d61c8c42d6812680bbfbea5c4b8d48d6792f0d24fe76e5f#0`

## Transaction Outputs

> Milestones 14 through 18 have been cancelled, each with a budget of 540,800 ada for a total of 2,704,000 ada. This amount should be returned to the treasury contract, while the remaining balance of 6,292,016 ada should be redirected back to the vendor contract.

Reviewer: ```suggestion
Milestones 14 through 18 have been cancelled, each with a budget of 540,800 ada for a total of 2,704,000 ada. This amount should be returned to the treasury contract, while the remaining balance of 6,292,016 ada should remain within vendor contract, with all other milestones.
```
